roboflow/models/classification.py

```diff
@@ -143,10 +143,11 @@ def __generate_url(self):
         # Generates URL based on all parameters
         splitted = self.id.rsplit("/")
         without_workspace = splitted[1]
+        version = self.version or splitted[2]
```

Comment:
do we enedd to check lenght here.  Cant see in this function what `self.id`.  looking at line 132 seems liek its setting `self.version` explicitly based on arg passed into `load_model`. If we're changing it, should we change it to take these via arguments instead?

Reply:
Good catch. Added the length check in there. Now version can come both in the id or the version parameter. If it comes in both, the version parameter overrides it.